crates/store/re_protos/src/v1alpha1/rerun.cloud.v1alpha1.ext.rs

```diff
@@ -1178,51 +1182,190 @@ pub struct RegisterWithDatasetTaskDescriptor {
+    pub fn fields() -> Vec<Field> {
+        vec![
+            Field::new(Self::PARTITION_ID, DataType::Utf8, false),
+            Field::new(
+                Self::LAYER_NAMES,
+                DataType::List(Self::layer_names_inner_field()),
+                false,
+            ),
+            Field::new(
+                Self::STORAGE_URLS,
+                DataType::List(Self::storage_urls_inner_field()),
+                false,
+            ),
+            Field::new(
+                Self::LAST_UPDATED_AT,
+                DataType::Timestamp(TimeUnit::Nanosecond, None),
+                false,
+            ),
+            Field::new(Self::NUM_CHUNKS, DataType::UInt64, false),
+            Field::new(Self::SIZE_BYTES, DataType::UInt64, false),
+        ]
+    }
+
+    pub fn schema() -> Schema {
+        Schema::new(Self::fields())
+    }
+
+    /// Helper to simplify instantiation of the dataframe in [`Self::data`].
+    pub fn create_dataframe(
+        partition_ids: Vec<String>,
+        layer_names: Vec<Vec<String>>,
+        storage_urls: Vec<Vec<String>>,
+        last_updated_at: Vec<i64>,
+        num_chunks: Vec<u64>,
+        size_bytes: Vec<u64>,
+    ) -> arrow::error::Result<RecordBatch> {
+        let row_count = partition_ids.len();
+        let schema = Arc::new(Self::schema());
+
+        let mut layer_names_builder =
+            ListBuilder::new(StringBuilder::new()).with_field(Self::layer_names_inner_field());
+
+        for mut inner_vec in layer_names {
+            for layer_name in inner_vec.drain(..) {
+                layer_names_builder.values().append_value(layer_name)
+            }
+            layer_names_builder.append(true);
+        }
+
+        let mut urls_builder =
+            ListBuilder::new(StringBuilder::new()).with_field(Self::storage_urls_inner_field());
+
+        for mut inner_vec in storage_urls {
+            for layer_name in inner_vec.drain(..) {
+                urls_builder.values().append_value(layer_name)
+            }
+            urls_builder.append(true);
+        }
+
+        let columns: Vec<ArrayRef> = vec![
+            Arc::new(StringArray::from(partition_ids)),
+            Arc::new(layer_names_builder.finish()),
+            Arc::new(urls_builder.finish()),
+            Arc::new(TimestampNanosecondArray::from(last_updated_at)),
+            Arc::new(UInt64Array::from(num_chunks)),
+            Arc::new(UInt64Array::from(size_bytes)),
+        ];
+
+        RecordBatch::try_new_with_options(
+            schema,
+            columns,
+            &RecordBatchOptions::default().with_row_count(Some(row_count)),
+        )
+    }
+
+    pub fn data(&self) -> Result<&DataframePart, TypeConversionError> {
+        Ok(self
+            .data
+            .as_ref()
+            .ok_or_else(|| missing_field!(Self, "data"))?)
+    }
+}
+
+// --- ScanDatasetManifestResponse --
+
+impl ScanDatasetManifestResponse {
+    pub const LAYER_NAME: &str = "rerun_layer_name";
```

Comment:
Similarly, I've been trying (and still failing) to make sure that all field constants are named `const FIELD_XXX` _and_ that they are named exactly the same as the string they represent so they A) can be lexically grepped for and B) can have their constant name guessed just by looking at any dataframe in the wild. :shrug: 
<img width="888" height="722" alt="image" src="https://github.com/user-attachments/assets/0ea81a38-28f6-4452-b7e1-0a2ef4037f07" />

Reply:
I'm gonna take the middle route here. I like the `FIELD_` prefix, but I'll drop the `RERUN_` part. It's noisy, and not having it seems not having still allows for the stated goals of this naming scheme.